### DISCUSSION TITLE
Cache images served when cache is disabled

### DISCUSSION THREAD

**Discussion by slowscape** [automadcms/discussions#19] [General]
I have never had cache turned on, but all images are still being served from the cache. 
Side note, probably should be a suggestion: Cache images pose an seo issue as the image names are changed to random characters.

**Comment by marcantondahmen** [comment]
The only caches you can turn off are the page cache and the content cache. Image caching has nothing to do with this and is always enabled. Resized images have to be served from somewhere and it would be to slow to resize them on the fly on every page load. 

However, as explained in the caching section of the dashboard, the image cache can be purged.

Also please not that this is a discussion forum. While this here is not a bug, bugs in general should be reported in the issues section of a project.
